WIRING.md

```diff
@@ -38,49 +38,64 @@ Pinout Teensy++ 2.0
-| 6              | A2              | 15               | PC2               |
-| 5              | A3              | 16               | PC3               |
-| 4              | A4              | 17               | PC4               |
-| 3              | A5              | 18               | PC5               |
-| 2              | A6              | 19               | PC6               |
-| 1              | A7              | 20               | PC7               |
-| 17             | D7              | 21               | PF7               |
-| 16             | D6              | 22               | PF6               |
-| 15             | D5              | 23               | PF5               |
-| 14             | D4              | 24               | PF4               |
-| 13             | D3              | 25               | PF3               |
-| 11             | D2              | 26               | PF2               |
-| 10             | D1              | 27               | PF1               |
-| 9              | D0              | 28               | PF0               |
-| -              | -               | 29               | AREF              |
-| 12             | GND             | 30               | GND               |
-| -              | -               | 31               | PE6               |
-| -              | -               | 32               | PE7               |
-| -              | -               | 33               | PB0               |
-| -              | -               | 34               | PB1               |
-| -              | -               | 35               | PB2               |
-| -              | -               | 36               | PB3               |
-| -              | -               | 37               | PB4               |
-| -              | -               | 38               | PB5               |
-| -              | -               | 39               | PB6               |
-| 23             | +5V             | 40               | VCC               |
+\* Signals A13, A14, A15 and R/W do not exist on the Atari 2600 cartridge slot, but are still handled by the Teensy dumper like they would exist. 
+This is done for easier adaptation to other platforms.
 
-(* These signals do not exist on the Atari 2600 cartridge slot, but are still handled by
-firmware like the would exist. For easier adaptation to other platforms.)
+\*\* Because GPIO pins are limited on the Nano, the cartridge A12 signal is wired to +5V, which basically always selects the ROM address space $1000-$1fff.\
+This means that when using the Nano dumper, only bankswitching schemes are supported which use hotspots *within* the cartridge ROM address space.
+For most cartridges brands (e.g. Atari, Activision, Imagic, CBS, M-Network and Parker Brothers) this is sufficient.
+If you need support for dumping TigerVision's 3F or Superbanking cartridges, please use the Teensy++ 2.0 microcontroller instead.
```

Comment:
Since A6 and A7 are also analog-in only, please mention that writing to the data bus is will be also not working as intended. (I'd really like to see how dumping a SARA SuperChip RAM cartridge works with the Nano, since there is a trick in dump_fx.c, line 27, where I put 0xFF on the databus, so that if the cartridge will return $FF for the RAM area).

Reply:
Good point; I'll add that note.
IIRC, just dumping a SARA cart will put whatever was on the bus into the RAM area, but didn't affect playing the game itself - I guess it doesn't matter, as the code in the game will probably clean it at start.
Btw: I think if you want the SARA rom files to be MD5-hash recognized by Stella, you should probably store $00 into the RAM area (not sure, but that is what I remember from looking into the binary data of commonly shared SARA roms)

Reply:
I've added the note about analog-in only A6 and A7 on the Nano in the WIRING.md